src/libterm/terminfo/parm.rs

```diff
@@ -437,31 +435,31 @@ impl Flags {
 
 #[derive(Copy, Clone)]
 enum FormatOp {
-    FormatDigit,
-    FormatOctal,
-    FormatHex,
-    FormatHEX,
-    FormatString,
+    Digit,
+    Octal,
+    Hex,
+    HEX,
```

Comment:
... Could we make these `LowerHex`/`UpperHex` if you're going to rename it anyway.